docs/moment/07-customization/08-am-pm.md

```diff
@@ -52,5 +50,3 @@ moment.updateLocale('en', {
     }
 });
 ```
-
-This has been deprecated. The **1.6.0** callback function syntax is now used instead.

```

Comment:
What does this refer to? If it's the example, we should remove the entire example.